barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.cpp

```diff
@@ -230,6 +230,9 @@ void build_constraints(Builder& builder,
         if (!constraint_system.avm_recursion_constraints.empty()) {
             info("WARNING: this circuit contains unhandled avm_recursion_constraints!");
         }
+        if (!constraint_system.ivc_recursion_constraints.empty()) {
```

Comment:
This means we now process ivc recursion constraints as part of the `build_constraints` like any other type of constraint